packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -47,7 +48,16 @@ const blockManagerConsole = anylogger('block-manager');
  */
 const getHostKey = path => `host.${path}`;
 
-async function buildSwingset(
+/**
+ * @param {Map<*, *>} mailboxStorage
+ * @param {*} bridgeOutbound
+ * @param {SwingStoreKernelStorage} kernelStorage
+ * @param {string} vatconfig absolute path
+ * @param {Record<string, any>} argv XXX argv should be an array but it's being called with object
+ * @param {{ ROLE: string }} env
+ * @param {*} options
+ */
+export async function buildSwingset(
```

Comment:
I really wasn't expecting this to be exported, and to stay an internal concern.

Reply:
I'm not clear on what the intent of this comment is. Do you want it to be factored differently? If so, how?

Reply:
I just have 3 pending PRs that are now failing/conflicting because of this change. I'm not sure if there is a better way to factor this, haven't looked closely yet. It's more of a process question, I wish I had known about this change coming.